src/easylink/step.py

```diff
@@ -483,8 +483,11 @@ class InputStep(IOStep):
 
     """
 
-    def __init__(self) -> None:
-        super().__init__(step_name="input_data", output_slots=(OutputSlot("all"),))
+    def __init__(
+        self,
+        output_slots: Iterable[OutputSlot] = (OutputSlot("all"),),
+    ) -> None:
+        super().__init__(step_name="input_data", output_slots=output_slots)
```

Comment:
Reverts a change inadvertently made in https://github.com/ihmeuw/easylink/commit/c0b8c4422dd54fc8c707023b10b14cd38fecd684 that made it impossible to create a pipeline schema with multiple user inputs.